src/SSHDebugPS/AD7Port.cs

```diff
@@ -49,10 +55,25 @@ public void EnsureConnected()
             GetConnection(ConnectionReason.Deferred);
         }
 
+        public bool IsConnected
+        {
+            get
+            {
+                return _connection != null;
+            }
+        }
+
         public int EnumProcesses(out IEnumDebugProcesses2 processEnum)
         {
             IEnumDebugProcesses2 result = null;
             var connection = GetConnection(ConnectionReason.Deferred);
+
+            if (connection == null)
+            {
+                processEnum = null;
+                return HR.E_REMOTE_CONNECT_USER_CANCELED;
```

Comment:
SDM currently doesn't suppress showing an error dialog for this value in this call as it does in other code paths. Added a work item for it.